orbitize/system.py

```diff
@@ -269,6 +313,11 @@ def compute_all_orbits(self, params_arr, epochs=None, comp_rebound=False):
                     radial velocities at each epoch.
 
         """
+
+        if use_c == None:
```

Comment:
Similar question here with the None vs default values for `use_c` and `use_gpu`. Perhaps we could just set one global variable in orbitize for these settings?

Reply:
Yeah, this is a good point. This code has already been merged into main like you said, so I say we open an issue and fix it once the open PRs are good to go.